unit_tests/test_config.py

```diff
@@ -288,6 +288,27 @@ def test_15_new_scylla_repo(self):
         conf.verify_configuration()
         self.assertEqual(conf.get('target_upgrade_version'), '2019.1.1')
 
+    def test_16_dictionary_config(self):
+        centos_repo = 'https://s3.amazonaws.com/downloads.scylladb.com/enterprise/rpm/unstable/centos/' \
+                      '9f724fedb93b4734fcfaec1156806921ff46e956-2bdfa9f7ef592edaf15e028faf3b7f695f39ebc1' \
+                      '-525a0255f73d454f8f97f32b8bdd71c8dec35d3d-a6b2b2355c666b1893f702a587287da978aeec22/71/scylla.repo'
+
+        os.environ['SCT_CLUSTER_BACKEND'] = 'gce'
+        os.environ['SCT_USER_PREFIX'] = 'testing'
+        os.environ['SCT_SCYLLA_REPO'] = centos_repo
+        os.environ['SCT_NEW_SCYLLA_REPO'] = centos_repo
+        os.environ['SCT_N_DB_NODES'] = '3'
+
+        os.environ['SCT_BACKUP_BUCKET_LOCATION'] = 'manager-backup-tests-eu-west-1'
```

Comment:
I thought the whole point was to test is, if it was a dict passed in `SCT_BACKUP_BUCKET_LOCATION`
this test doesn't do anything now, just reads from the default config